Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class MakeStuffWithInput
+    {
+        //TODO Make the Stringmakers more streamlined so they can be called and read more easy. and modular.
+
+        /// <summary>
+        /// This Gives back a string that reads: people are/is the richest people. ¤ amount. 
+        /// </summary>
+        /// <param name="numberOfPeople"></param>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringRichestPeople(int numberOfPeople, string people, decimal amount)
+        {
+            if (numberOfPeople > 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)